slapd-cli

```diff
@@ -2027,6 +2028,14 @@ userPassword: ${DATA_USER_PW_HASH}" >> ${DATA_FILE_PATH}
 
 }
 
+removeoldbackups() {
+        if [ "x$BACKUP_DELETE_AFTER_DAYS" != "x" ]
+        then
+                message "info" "[INFO] Removing backup files older than $BACKUP_DELETE_AFTER_DAYS days"
+                find $BACKUP_PATH -mtime +$BACKUP_DELETE_AFTER_DAYS -exec rm {} \;
```

Comment:
1/ It would be more efficient if we write `-delete`

2/ Also, maybe we should document this caveat:
- admin creates one backup day 1, then another backup day 2
- admin edit the first backup
- admin create another backup. Backup 2 is deleted because it is older in term of modification, but newer in term of backup generation.

3/ Last but not least: I think we shouldn't remove indifferently config and data backups.

A configuration backup can be done one time and stay on the server indefinitely. Maybe we could have differentiate parameters:

```
BACKUP_DATA_DELETE_AFTER_DAYS
BACKUP_CONFIG_DELETE_AFTER_DAYS
```

and target the corresponding files when deleting?

Reply:
> 1/ It would be more efficient if we write -delete
I don't understand

> 2/ Also, maybe we should document this caveat...
Yes we can add a warning in the documentation, but this is the same issue with the restore function

> 3/ Last but not least: I think we shouldn't remove indifferently config and data backups
Complex to find the files that are the data, unless we reproduce the loop on databases. It could maybe be an enhancement for further release?

Reply:
>>1/ It would be more efficient if we write -delete
>I don't understand
done in 8994e83